js/bitget3.js

```diff
@@ -0,0 +1,2961 @@
+            'info': order,
+            'id': id,
+            'clientOrderId': clientOrderId,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'lastTradeTimestamp': undefined,
+            'symbol': symbol,
+            'type': type,
+            'timeInForce': undefined,
+            'postOnly': undefined,
+            'side': side,
+            'price': price,
+            'stopPrice': undefined,
+            'average': average,
+            'cost': cost,
+            'amount': amount,
+            'filled': filled,
+            'remaining': undefined,
+            'status': status,
+            'fee': fee,
+            'trades': undefined,
+        }, market);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        await this.loadAccounts ();
+        const market = this.market (symbol);
+        //
+        // spot
+        //
+        //     account_id true string Account ID, obtained using the accounts method. Currency transactions use the accountid of the'spot' account; for loan asset transactions, please use the accountid of the'margin' account
+        //     amount true string A limit order indicates the quantity of the order, when a market price buy order indicates how much money to buy, and when a market price sell order indicates how much currency to sell
+        //     price false string Order price, market order does not pass this parameter
+        //     source false string Order source api
+        //     symbol true string Trading pair  btc_usdt, eth_btc ...
+        //     type true string Order Type  buy-market: buy at market price, sell-market: sell at market price, buy-limit: buy at limit price, sell-limit: sell at limit price
+        //
+        // swap
+        //
+        //     symbol String Yes Contract ID
+        //     client_oid String Yes customize order IDs to identify your orders. (Less than 50 characters without special characters,
+        //     size String Yes Quantity to buy or sell (value not equal to 0 or negative)
+        //     type String Yes 1 Open long 2Open short 3 Close long 4 Close short
+        //     order_type String Yes 0: Normal order (Unfilled and 0 imply normal limit order) 1: Post only 2: Fill or Kill 3: Immediate Or Cancel
+        //     match_price String Yes 0 Limit price 1 market price
+        //     price String No Price of each contract
+        //
+        const request = {
+            'symbol': market['id'],
+        };
+        const clientOrderId = this.safeString2 (params, 'client_oid', 'clientOrderId', this.uuid ());
+        params = this.omit (params, [ 'client_oid', 'clientOrderId' ]);
+        let method = undefined;
+        if (market['spot']) {
+            const accountId = await this.getAccountId ({
+                'type': market['type'],
+            });
+            method = 'apiPostOrderOrdersPlace';
```

Comment:
could we use the latest method [`spotPostTradeOrders`](https://bitgetlimited.github.io/apidoc/en/spot/#place-order) since it also supports `clientOrderId` param